src/fiatconnect/saga.ts

```diff
@@ -893,97 +893,128 @@ export function* handleFetchFiatConnectProviders() {
   }
 }
 
-export function* handleCreateFiatConnectTransfer({
+/**
+ * Initiates a fiatconnect transfer using the fiatconnect SDK to call a transfer/out or transfer/in endpoint
+ * for a provider.
+ **/
+export function* _initiateFiatConnectTransferWithProvider({
   payload: params,
 }: ReturnType<typeof createFiatConnectTransfer>) {
   const { flow, fiatConnectQuote, fiatAccountId } = params
   const quoteId = fiatConnectQuote.getQuoteId()
+  const transferType = flow === CICOFlow.CashIn ? 'transferIn' : 'transferOut'
```

Comment:
nit: maybe call this `transferFn` or `transferFnName`? Took me a couple of reads of the call method below to understand this was the function name